File: JAVASCRIPT/kuantum_kaos_yoneticisi_js/models/VeriPaketi.js
// models/VeriPaketi.js
const KuantumNesnesi = require('./KuantumNesnesi');

class VeriPaketi extends KuantumNesnesi {
    constructor(id) {
        super(id, 1, 80); 
    }

    AnalizEt() {
        this._azaltVeKontrolEt(5);
        console.log(`[VeriPaketi - ${this.ID}] Veri içeriği okundu. Stabilite -5.`);
    }
}
module.exports = VeriPaketi;

File: JAVA/KuantumKaosYoneticisiJava/src/models/VeriPaketi.java
package models;

public class VeriPaketi extends KuantumNesnesi {
    
    public VeriPaketi(String id) {
        super(id, 1, 80); 
    }

    @Override
    public void AnalizEt() {
        azaltVeKontrolEt(5); // Stabilite 5 birim düşer.
        System.out.println("[VeriPaketi - " + getID() + "] Veri içeriği okundu. Stabilite -5.");
    }
}